app/views/charge.njk

```diff
@@ -6,8 +6,13 @@
   {{ __("cardDetails.title") }}
 {% endblock %}
 
+{% set bodyClasses = "google-pay-unavailable" %}
 {% set mainClasses = "charge-new" %}
 
+{% block bodyStart %}
```

Comment:
If there `apple-pay-unavailable` weren’t a thing, this code could be wrapped in an `{ %if allowApplePay %}` block

Reply:
Allowing Apple Pay and Apple Pay being unavailable are different things

Reply:
Yes but what I meant was that we don’t need to check if Apple Pay is available if it’s not allowed in the first place